docs/docs/developers/guides/js_apps/pay_fees.md

```diff
@@ -150,6 +162,18 @@ import { SponsoredFeePaymentMethod } from "@aztec/aztec.js/fee/testing";
 const paymentMethod = new SponsoredFeePaymentMethod(deployedSponsoredFPC);
 ```
 
+Register the SponsoredFPC in the PXE:
+
+```ts
+import { SponsoredFPCContract } from "@aztec/noir-contracts.js/SponsoredFPC";
```

Comment:
What is the difference with import FPCContract from FPC above (line 131), vs this line?
Can these be code snippets from somewhere?

Reply:
This one is for `SponsoredFPCContracts`, the previous is just for `FPCContracts`. I searched e2e tests for code that would work out of context of the tests, but didn't find any.